src/molecules/list/component.tsx

```diff
@@ -67,7 +67,7 @@ export const List = (props: IListProps): React.ReactElement => {
               key={child.props.itemKey}
               id={child.props.id}
               className={child.props.className}
-              theme={props.theme?.listItemTheme}
+              $theme={props.theme?.listItemTheme}
```

Comment:
ah here it should stay as theme, its only the internally passed things that convert to using $